contrib/ci/dependencies.xml

```diff
@@ -1754,4 +1754,16 @@
       <package variant="mingw64">msitools</package>
     </distro>
   </dependency>
+  <!-- optional -->
+  <dependency id="rust">
+    <distro id="fedora">
+      <package variant="x86_64" />
+    </distro>
+    <distro id="debian">
+      <package variant="x86_64" />
+    </distro>
+    <distro id="ubuntu">
+      <package variant="x86_64" />
```

Comment:
for these I think it's [rustc](https://tracker.debian.org/pkg/rustc) that you're looking for.

Reply:
I think I can use rustc for Fedora too; Fixed.